station.go

```diff
@@ -417,3 +435,44 @@ func (sd *schemaDetails) validateProtoMsg(msg any) ([]byte, error) {
 
 	return msgBytes, nil
 }
+
+func (sd *schemaDetails) validJsonSchemaMsg(msg any) ([]byte, error) {
+	var (
+		msgBytes []byte
+		err      error
+		message  interface{}
+	)
+	switch msg.(type) {
+	case []byte:
+		msgBytes = msg.([]byte)
+		if err := json.Unmarshal(msgBytes, &message); err != nil {
+			return nil, memphisError(err)
+		}
+		break
+	case interface{}:
```

Comment:
`interface{}` and `any` are the same, so every `msg` that is not of type `[]byte` will match here, so we can either remove the `default:` or the `case interface{}:`

or just use the `reflect.TypeOf(msg)` without the `switch msg.(type)`